swagger_server/itm/itm_action_handler.py

```diff
@@ -52,16 +52,18 @@ def _reveal_injuries(self, source: Character, target: Character):
             target.unstructured = source.unstructured_postassess
 
     def __successful_treatment(self, treatment: str, injury_name: str, location: str) -> bool:
-        # NOTE: Asthmatic, Forehead Scrape, Ear Bleed, an Internal injuries are currently untreatable.
+        # NOTE: Asthmatic, Forehead Scrape, Ear Bleed, Traumatic Brain Injury, Open Abdominal Wound and Internal injuries are currently untreatable.
         # This logic is in sync with the current OSU Simulator, but may diverge at a later date.
         """
             Head Injuries
             Forehead Scrape (Abrasion): Pressure bandage
             Face Shrapnel: Nasopharyngeal airway
             Ear Bleed: None
+            Traumatic Brain Injury: None
 
             Neck Injuries
             Neck Puncture: Hemostatic gauze
+            Neck Burn: Burn Dressing 
```

Comment:
I hadn't added Burn locations to these comments, but it's a good idea.  Please add for:
- Arm Injuries -> Forearm Burn
- Leg Injuries -> Calf Burn
- Leg Injuries -> Thigh Burn
- Chest Injuries -> Chest Burn

Also, please remove "Broken Forearm" and "Broken Wrist".